### PR TITLE
Give access to attributes of stdin and stdout that don't exist in the file object

### PR DESCRIPTION
**Pull request by fuxpavel**
Attributes like "encoding" are needed in pieces of code that is debugged and fails
the code during the debugging

### CODE REVIEW

File: rpdb/__init__.py [hunk 1]
```diff
@@ -8,6 +8,21 @@
 import threading
 import sys
 import traceback
+import exceptions
+
```
Comment (tamentis) [inline rpdb/__init__.py]: Please use two new-lines (PEP-8), and also I don't think the `import exceptions` part will work on Python 3, you can probably raise AttributeError directly.

File: rpdb/__init__.py [hunk 2]
```diff
@@ -35,7 +50,9 @@ def __init__(self, addr="127.0.0.1", port=4444):
 
         (clientsocket, address) = self.skt.accept()
         handle = clientsocket.makefile('rw')
-        pdb.Pdb.__init__(self, completekey='tab', stdin=handle, stdout=handle)
+        pdb.Pdb.__init__(self, completekey='tab',
+                                 stdin=FileObjectWrapper(handle, self.old_stdin),
+                                 stdout=FileObjectWrapper(handle, self.old_stdin))
```
Comment (tamentis) [inline rpdb/__init__.py]: Also, please align the wrapped line with `self`.

**Comment by tamentis** [comment]
Thanks for the Pull Request, I added a couple comments in-line but beside that it looks good.

**Comment by fuxpavel** [comment]
I have fixed your comments.

**Pull request merged**